Input phone number

GusBaamonde opened the pull request:
- Add flags image, style and js files for new intel-input library
- https://cdn.fromdoppler.com/doppler-ui-library/build165/signup.html

![tel_input_validate](https://user-images.githubusercontent.com/46735526/55355141-e203a880-549d-11e9-9430-c1ab8fa43bba.gif)

cbernat commented on assets/scss/libs/intlTelInput/_intlTelInput.scss:
```diff
@@ -76,11 +76,13 @@ font-family: $dp-font-family;
     .intl-tel-input .country-list .country.highlight {
-      background-color: rgba(0, 0, 0, 0.05); }
+      background-color: rgba(0, 0, 0, 0.05);
+font-size: $dp-font-size-form;
```
Why is this so badly formatted? is it autogenerated?

cbernat commented:
I've added some comments, it's looking very good!

cbernat commented on assets/js/intlTelInput.min.js:
```diff
@@ -0,0 +1,1201 @@
+  /*
```
You can add this library also via npm: https://www.npmjs.com/package/intl-tel-input I think it would be more cleaner, what do you think?

GusBaamonde replied:
cool! I try with this

IgnacioRodrigues commented on assets/js/main.js:
```diff
@@ -69,3 +69,60 @@ $(".show-hide").click(function() {
+  ];
+
+// initialise plugin
+
```
Delete this space

IgnacioRodrigues commented on assets/js/main.js:
```diff
@@ -69,3 +69,60 @@ $(".show-hide").click(function() {
+      if (iti.isValidNumber()) {
+        $(inputPhone).closest('.field-item').removeClass('error');
+      } else {
+
```
Delete this space

andresmoschini commented on assets/templates/signup.html:
```diff
@@ -1,155 +1,224 @@
+    </main>
+    <!-- build:js:dist js/app.js -->
+    <link rel="stylesheet" type="text/css" href="https://intl-tel-input.com/node_modules/intl-tel-input/build/css/intlTelInput.css">
+    <style> .intl-tel-input.separate-dial-code .selected-dial-code {
```
What are these styles?

GusBaamonde replied:
first there is the call of the style sheet of the library intl-tel-input, and then I overwrite two classes to be able to modify details.

andresmoschini replied:
Solo para confirmar, ¿no deberíamos incluir esos estilos que sobreescribimos como parte de la librería? 

No estoy haciendo una apreciación, solo preguntando.

GusBaamonde replied:
Mmmm, yo creo que no. Si esto se incluye como un componente react seguramente tendremos que incluir los estilos dentro del componente y ahí darle los estilos correctos y finales.
Yo creo que si lo metemos en la librería, deberíamos meter TODO incluso los js.

GusBaamonde commented:
- I added external scripts and css for a correct visualization.
- I deleted all the js and css files included.
- this component will be added by react

https://www.npmjs.com/package/react-intl-tel-input

andresmoschini commented:
:tada: This PR is included in version 1.0.0 :tada:

The release is available on [GitHub release](https://github.com/FromDoppler/Doppler-UI-System/releases/tag/v1.0.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
